src/main/kotlin/io/snyk/plugin/settings/SnykProjectSettingsConfigurable.kt

```diff
@@ -98,8 +99,13 @@ class SnykProjectSettingsConfigurable(val project: Project) : SearchableConfigur
             snykProjectSettingsService?.additionalParameters = snykSettingsDialog.getAdditionalParameters()
         }
 
-        val params = DidChangeConfigurationParams(LanguageServerWrapper.getInstance().getSettings())
-        LanguageServerWrapper.getInstance().languageServer.workspaceService.didChangeConfiguration(params)
+        val wrapper = LanguageServerWrapper.getInstance()
+        val params = DidChangeConfigurationParams(wrapper.getSettings())
+        wrapper.languageServer.workspaceService.didChangeConfiguration(params)
+
+        runBackgroundableTask("Updating Snyk Code settings", project, true) {
+            settingsStateService.isGlobalIgnoresFeatureEnabled = wrapper.getFeatureFlagStatus("snykCodeConsistentIgnores")
```

Comment:
Here the guard is missing that you only call LS when `isSnykCodeLSEnabled() == true`

Reply:
Done, please double check 🙏